### PR TITLE
Trello customizations of library

### PR DESCRIPTION
**Pull request by dlew**
I'm just porting out customizations into bypass. These all existed before, but I figured this would be a good sanity check.

### CODE REVIEW

**Comment by ulyssesdotcodes** [comment]
We don't have List types?

**Comment by dlew** [comment]
We only do special things for the bullet points themselves, not for the lists.

File: platform/android/library/src/in/uncod/android/bypass/style/CodeBackgroundSpan.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+        mPaint = new Paint();
+        mPaint.setColor(GRAY);
+    }
+
+    @Override
+    public void drawBackground(
+        Canvas c, Paint p, int left, int right, int top, int baseline, int bottom, CharSequence text, int start,
+        int end, int lnum) {
+        c.drawRect(left + mIndent - 4, top - 4, right, bottom, mPaint);
```
Comment (ulyssesdotcodes) [inline platform/android/library/src/in/uncod/android/bypass/style/CodeBackgroundSpan.java:30]: Where does the 4 come from?

Comment (dlew) [inline platform/android/library/src/in/uncod/android/bypass/style/CodeBackgroundSpan.java:30]: Lots of trial and error and experimentation, IIRC.

I'm willing to switch this up and actually use dp at some point, but it's been like this for at least a year.

Comment (dlew) [inline platform/android/library/src/in/uncod/android/bypass/style/CodeBackgroundSpan.java:30]: Made this to clear it up later: https://trello.com/c/qO93Z384/3370-use-dips-in-codebackgroundspan

Comment (ulyssesdotcodes) [inline platform/android/library/src/in/uncod/android/bypass/style/CodeBackgroundSpan.java:30]: Sounds good.

**Comment by ulyssesdotcodes** [comment]
:+1:

**Pull request merged**